components/provisioner/go.mod

```diff
@@ -103,20 +103,26 @@ require (
 	github.com/xeipuuv/gojsonreference v0.0.0-20180127040603-bd5ef7bd5415 // indirect
 	github.com/xeipuuv/gojsonschema v1.2.0 // indirect
 	go.opencensus.io v0.23.0 // indirect
-	golang.org/x/crypto v0.6.0 // indirect
-	golang.org/x/mod v0.9.0 // indirect
-	golang.org/x/net v0.8.0 // indirect
-	golang.org/x/oauth2 v0.0.0-20220411215720-9780585627b5 // indirect
-	golang.org/x/sys v0.6.0 // indirect
-	golang.org/x/term v0.6.0 // indirect
-	golang.org/x/text v0.8.0 // indirect
+	golang.org/x/crypto v0.14.0 // indirect
+	golang.org/x/exp v0.0.0-20231006140011-7918f672742d // indirect
+	golang.org/x/image v0.13.0 // indirect
+	golang.org/x/lint v0.0.0-20210508222113-6edffad5e616 // indirect
+	golang.org/x/mobile v0.0.0-20231006135142-2b44d11868fe // indirect
```

Comment:
I don't think that this package (for mobile deployment e.g. Android or iOS) is required in Provisioner

Reply:
<img width="871" alt="image" src="https://github.com/kyma-project/control-plane/assets/9433601/9821c421-4c6f-4a1d-a3cf-e04e12c47543">
it is already there on `main`
[link](https://github.com/kyma-project/control-plane/blob/70ab8deac81690f35b13446d3fcee31b2aa39789/components/provisioner/go.sum#L852-L853)

Reply:
But I absolutely agree, I don't think `provisioner` supports mobile in any way

Reply:
and it's required